components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -21,7 +21,8 @@ EKS_CLUSTER_VERSION=${EKS_CLUSTER_VERSION:-"1.15"} # EKS cluster K8s version
 EKS_NODE_COUNT=${EKS_NODE_COUNT:-"1"} # The initial node count of the EKS cluster
 EKS_PUBLIC_SUBNETS=${EKS_PUBLIC_SUBNETS:-""}
 EKS_PRIVATE_SUBNETS=${EKS_PRIVATE_SUBNETS:-""}
-SKIP_OIDC_SETUP=${SKIP_OIDC_SETUP:-"false"}
+SKIP_KFP_OIDC_SETUP=${SKIP_KFP_OIDC_SETUP:-"false"}
+KFP_VERSION=${KFP_VERSION:-"=0.5.1"}
```

Comment:
Oops you included the equals sign here accidentally?

Reply:
Nice catch !

(running integ tests btw)